packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item.tsx

```diff
@@ -82,7 +96,7 @@ export class SegmentedControlItem {
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
-      <button type="button" {...getButtonAttributes(this.host.selected, this.disabled)}>
+      <button type="button" {...getSegmentedControlItemAriaAttributes(this.host.selected, this.disabled, this.aria)}>
```

Comment:
Should we validate if an `aria-label` is set when no content is passed?

Reply:
@henrijoss good approach, but we should think about validating against accessible names in a more generic way instead of adding custom validations for every use case (also this validation is missing on some other components, like link, button, etc.)

Google provides some (unofficial) package for calculating an HTML element's accessible name: https://github.com/google/accname 

But this would be a separate task. What do you think?

Reply:
Don't know about the package but I also think a more generic way of making sure the accessible names are provided would be good. Do we already have a ticket for this?

Reply:
Nope, but I will add one :-)